### DISCUSSION TITLE
同步阿里云报错，报错内容见图

### DISCUSSION THREAD

**Discussion by yaoxiao99** [remotely-save/remotely-save#461] [Q&A] [answered]
<img width="304" alt="Snipaste_2024-01-22_18-33-08" src="https://github.com/remotely-save/remotely-save/assets/31535701/d112bdbe-d498-4966-9ba0-6bccae605c88">

remotely-save-1705919640134:中断同步同步来源=manual，出错阶段5=getting_remote files list

Invalid region: region was not a valid DNSname.

**Comment by fyears** [comment]
将类似 oss-cn-hangzhou.aliyuncs.com 的改成 oss-cn-hangzhou

**Reply by yaoxiao99** [reply]
改了，还是报错。同样的错误
服务器地址=oss-cn-shenzhen
区域=oss-cn-shenzhen-internal.aliyuncs.com
![Uploading Snipaste_2024-01-23_10-07-32.png…]()

**Comment by yaoxiao99** [comment]
改了，还是报错。同样的错误
服务器地址=oss-cn-shenzhen
区域=oss-cn-shenzhen-internal.aliyuncs.com
![Uploading Snipaste_2024-01-23_10-07-32.png…]()

**Reply by fyears** [reply, marked as answer]
搞反了。正确的是服务器地址更长：

服务器地址= oss-cn-shenzhen-internal.aliyuncs.com 或者 oss-cn-shenzhen.aliyuncs.com 

区域= oss-cn-shenzhen
